scripts/upload_zoom_recordings.py

```diff
@@ -68,6 +69,7 @@ def download_file(url, download_path):
                     command = [
```

Comment:
Whoa! Great timing dc. 11 hours ago someone did some of the work for us ;)
https://github.com/tokland/youtube-upload/issues/194

I'll create a new ticket to track that, and maybe do the work of the PR on `youtube-upload` when I get a sec..!